src/build/build.ts

```diff
@@ -115,7 +115,7 @@ export async function build(options: BuildOptions, config: ProjectConfig): Promi
 
   // Once the unbundled build stream is complete, create a service worker for the build
   let unbundledPostProcessing = Promise.all([loadSWConfig, waitFor(unbundledPhase)]).then((results) => {
-    let swConfig: SWConfig = results[0];
+    let swConfig: SWConfig|undefined = results[0] || undefined;
```

Comment:
The stable compiler doesn't complain, but this code is more correct than before. `parsePreCacheConfig` returns `Promise<SWConfig|null>` and `addServiceWorker` wants `SWConfig|undefined`